spec/ameba/rule/lint/literals_comparison_spec.cr

```diff
@@ -41,26 +41,13 @@ module Ameba::Rule::Lint
         CRYSTAL
     end
 
-    it "reports if there is a static path comparison evaluating to false" do
-      expect_issue subject, <<-CRYSTAL
-        String == Nil
-        # ^^^^^^^^^^^ error: Comparison always evaluates to false
-        CRYSTAL
-    end
-
     context "macro" do
-      pending "reports in macro scope" do
+      it "reports in macro scope" do
         expect_issue subject, <<-CRYSTAL
           {{ "foo" == "foo" }}
            # ^^^^^^^^^^^^^^ error: Comparison always evaluates to true
           CRYSTAL
       end
-
-      it "passes for free variables comparisons in macro scope" do
-        expect_no_issues subject, <<-CRYSTAL
-          {{ T == Nil }}
-          CRYSTAL
-      end
```

Comment:
sorry, unclear why is this test removed if that's the catching regression scenario reported in https://github.com/crystal-ameba/ameba/issues/295

Reply:
It's because the support for paths has been removed altogether, so all of the `String == Nil` type of comparisons won't be reported at all, neither in macro context nor in regular crystal code.